ksa_compliance/ksa_compliance/doctype/sales_invoice_additional_fields/sales_invoice_additional_fields.py

```diff
@@ -351,16 +349,18 @@ def _add_integration_log_document(self, zatca_message, integration_status, zatca
         integration_doc.insert(ignore_permissions=True)
 
     @property
-    def qr_image(self) -> str | None:
+    def qr_image_uri(self) -> str | None:
         if not self.qr_code:
             return None
         qr = pyqrcode.create(self.qr_code)
         with BytesIO() as buffer:
             qr.png(buffer, scale=7)
             buffer.seek(0)
-            img_str = base64.b64encode(buffer.getvalue()).decode("utf-8")
-            return img_str
+            return "data:image/png;base64," + base64.b64encode(buffer.getvalue()).decode("utf-8")
 
+    @property
+    def qr_code_image(self):
+        return self.qr_code_image
```

Comment:
After testing, this part is not required to the image to be shown correclty in the desk, it could be removed, because it has already the src in its options.

Reply:
It's not about need. This appears to me to be broken. Calling it would result in an infinite loop that eventually terminates with a `RecursionError`